streampipes-client-python/streampipes/model/common.py

```diff
@@ -73,17 +87,19 @@ class EventProperty(BasicModel):
     Data model of an `EventProperty` in compliance to the StreamPipes Backend.
     """
 
-    class_name: Optional[StrictStr] = Field(alias="@class")
-    element_id: Optional[StrictStr]
+    class_name: StrictStr = Field(
+        alias="@class", default_factory=lambda: "org.apache.streampipes.model.schema.EventPropertyPrimitive"
+    )
+    element_id: StrictStr = Field(default_factory=lambda: f"sp:eventproperty:{random_letters(6)}")
     label: Optional[StrictStr]
     description: Optional[StrictStr]
     runtime_name: StrictStr
-    required: StrictBool
-    domain_properties: List[StrictStr]
-    property_scope: Optional[StrictStr]
-    index: StrictInt
+    required: StrictBool = False
```

Comment:
it would be beneficial to use the pydantic functionality here, so in this case `Field(default=False)`
Some applies for the upcoming rows, for an empty array you can use the `default_factory`